examples/metronome.js

```diff
@@ -1,19 +1,44 @@
 var bap = require('../index');
```

Comment:
I think we should keep the original metronome example, it's a really simple "hello world" for anyone looking at the library. Add another example to showcase the custom waveshape, maybe with something that showcases the benefit of a custom waveshape more.